services/backend/src/core/profile/profile.js

```diff
@@ -1044,96 +1044,103 @@ async function getPublicProfileById(request, response) {
         language
       );
 
-      const isFriends = result.friends.some((item) => item.id === userId);
-
-      if (
-        result.visibleCards.manager === "PRIVATE" ||
-        (result.visibleCards.manager === "FRIENDS" && !isFriends)
-      ) {
-        delete result.manager;
-      }
+      const isConnection = result.connections.some(
+        (item) => item.id === userId
+      );
 
       if (
         result.visibleCards.info === "PRIVATE" ||
-        (result.visibleCards.info === "FRIENDS" && !isFriends)
+        (result.visibleCards.info === "CONNECTIONS" && !isConnection)
       ) {
```

Comment:
For all of these big if statements, since they all have the "same" logic, we could create a function that just receives the visibleCards key

something like this just above the first if statement?
```js
const hideCard = (key) => result.visibleCards[key] === "PRIVATE" || (result.visibleCards[key] === "CONNECTIONS" && !isConnection);
```
Then the if statements could look a bit cleaner
```js
if (hideCard("info")) { .... }
```